### DISCUSSION TITLE
Move community server to revolt.chat instead of discord to help with privacy concerns

### DISCUSSION THREAD

**Discussion by FillingTheVoidOnYT** [JulianPrieber/littlelink-custom#183] [Issues]
title

**Comment by JulianPrieber** [comment]
@MagicLike

**Comment by MagicLike** [comment]
@FillingTheVoidOnYT
Is there some kind of comparison to Discord? What features do we get and which are not implemented?
Are there bots, which we can use?
Are there threads?
Are there announcement channels?

**Comment by FillingTheVoidOnYT** [comment]
> @FillingTheVoidOnYT
> Is there some kind of comparison to Discord? What features do we get and which are not implemented?
> Are there bots, which we can use?
> Are there threads?
> Are there announcement channels?

Everything discord has plus encryption on the way 
There are many bots (automod, pins, rss)
There are message threads in the works 
There also is announcement channels

**Comment by FillingTheVoidOnYT** [comment]
Here’s their roadmap 
https://trello.com/b/4e2O7tge/roadmap

**Comment by MagicLike** [comment]
I have just made a quick account and my first impressions are great....
I wouldn't say, that there are "many" bots, but a good amount.

> Here’s their roadmap https://trello.com/b/4e2O7tge/roadmap

I'd wait with the move, because there is this card: https://trello.com/c/xiTaKb2b/27-discord-compatibility-layer
Maybe the transfer will be easier then...

**Comment by FillingTheVoidOnYT** [comment]
> I have just made a quick account and my first impressions are great.... I wouldn't say, that there are "many" bots, but a good amount.
> 
> > Here’s their roadmap https://trello.com/b/4e2O7tge/roadmap
> > I'd wait with the move, because there is this card: https://trello.com/c/xiTaKb2b/27-discord-compatibility-layer
> > Maybe the transfer will be easier then...

yea maybe not as much as discord but plenty of bots for an open source project, it has seen great amounts of growth though daily hundreds are adopting tho. There is no analytics for the developer so they don't know an exact number but just my servers have hundreds of people and people joining daily that.

**Comment by MagicLike** [comment]
@JulianPrieber I will just go ahead and create a LLC server to test if it fits...

**Reply by FillingTheVoidOnYT** [reply]
send an invite

**Reply by MagicLike** [reply]
First of all, sorry for not replying. Here are my "test" results:
As things stand, it's not a 100% fit for our purpose. It lacks certain features and we would also have to program our own bot to get GitHub updates and other things....
But I'll look at the next updates and maybe it will be better. Don't get me wrong, it's a great project, but there are some things that make it not *the* Discord replacement.
Please let me point out that [Techlore also uses and likes Discord](https://invidious.snopyta.org/watch?v=oHCK3NKFBW4) - under certain circumstances - which is exactly what we use it for.
